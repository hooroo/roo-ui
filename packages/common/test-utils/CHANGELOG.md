# Change Log

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

<a name="0.4.0"></a>
# [0.4.0](https://github.com/hooroo/roo-ui/compare/v0.3.3...v0.4.0) (2018-04-26)


### Features

* add theming ([bee896a](https://github.com/hooroo/roo-ui/commit/bee896a))
