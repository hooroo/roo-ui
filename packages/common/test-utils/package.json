{
  "name": "@roo-ui/test-utils",
  "version": "0.0.0",
  "main": "dist/index.js",
  "publishConfig": {
    "access": "public"
  },
  "scripts": {
    "build": "babel src -d dist --copy-files"
  }
}
