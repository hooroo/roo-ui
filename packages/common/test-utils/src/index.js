export { default as shallowWithTheme } from './shallowWithTheme'; // eslint-disable-line import/prefer-default-export
