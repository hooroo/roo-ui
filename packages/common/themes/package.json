{
  "name": "@roo-ui/themes",
  "version": "0.4.0",
  "main": "dist/index.js",
  "publishConfig": {
    "access": "public"
  },
  "scripts": {
    "build": "babel src -d dist --copy-files"
  }
}
