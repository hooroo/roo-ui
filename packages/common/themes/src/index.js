import qantas from './qantas';

export { qantas }; // eslint-disable-line import/prefer-default-export
