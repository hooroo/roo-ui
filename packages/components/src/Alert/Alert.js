import React from 'react';
import PropTypes from 'prop-types';
import cleanElement from 'clean-element';

import { Container, NakedButton, Text, Flex, Box, Icon } from '..';

const CleanDiv = cleanElement('div');

const Base = ({
  children,
  icon,
  onClose,
  ...props
}) => (
  <CleanDiv {...props}>
    <Container>
      <Flex py={4}>
        {icon && (
          <Box pl={1} pr={3}>
            <Icon {...icon} />
          </Box>
        )}

        <Text mb={0} textAlign="left">
          {children}
        </Text>

        {onClose && (
          <Box ml="auto" pr={1} pl={3}>
            <NakedButton onClick={onClose}>
              <Icon name="close" />
            </NakedButton>
          </Box>
        )}
      </Flex>
    </Container>
  </CleanDiv>
);

Base.propTypes = {
  children: PropTypes.node.isRequired,
  icon: PropTypes.shape(Icon.propTypes),
  onClose: PropTypes.func,
};

Base.defaultProps = {
  icon: null,
  onClose: null,
};

const Alert = Box.withComponent(Base);
Alert.defaultProps = {
  ...Box.defaultProps,
  bg: 'grey.3',
  mb: 3,
};

Alert.info = Alert.extend``;
Alert.info.displayName = 'Alert';
Alert.info.defaultProps = {
  ...Alert.defaultProps,
  bg: 'ui.infoBackground',
  icon: {
    name: 'info',
    color: 'grey.0',
  },
};

Alert.success = Alert.extend``;
Alert.success.displayName = 'Alert';
Alert.success.defaultProps = {
  ...Alert.defaultProps,
  bg: 'ui.successBackground',
  icon: {
    name: 'checkCircle',
    color: 'ui.success',
  },
};

Alert.error = Alert.extend``;
Alert.error.displayName = 'Alert';
Alert.error.defaultProps = {
  ...Alert.defaultProps,
  bg: 'ui.errorBackground',
  icon: {
    name: 'warning',
    color: 'ui.error',
  },
};

export default Alert;
