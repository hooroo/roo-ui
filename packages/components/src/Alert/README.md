# Alert

<!-- STORY -->

## Installation

```shell
$ yarn add @roo-ui/components
```

## Example

```js
import { Alert } from '@roo-ui/components';

export default (
  <Alert onClose={closeAlert} icon={{ name: 'hotel' }}>
    Hello world
  </Alert>
);
```

## Properties

| Name        | Description                                                          | Type     | Default   | Required? |
|-------------|----------------------------------------------------------------------|----------|-----------|-----------|
| `onClose`   | called when close icon is clicked                                    | `func`   | -         | -         |
| `contained` | wrap the contents of the <Alert /> in a <Container />                | `bool`   | false     | -         |
| `icon`      | render an icon                                                       | `shape`  | -         | -         |
| `variant`      | `alertStyle` from theme. One of `default`, `info`, `success` or `error` | `string` | 'default' | -         |

Refer to [<Icon />](../Icon) for available `icon` properties.
