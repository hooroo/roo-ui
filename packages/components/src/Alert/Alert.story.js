import React from 'react';
import { storiesOf } from '@storybook/react';
import { withDocs } from 'storybook-readme';
import { boolean } from '@storybook/addon-knobs/react';

import Alert from './Alert';
import README from './README.md';

storiesOf('Components|Alert', module)
  .addDecorator(withDocs(README))
  .add('<Alert />', () => (
    <Alert contained={boolean('Contained', false)} icon={{ name: 'hotel' }}>
      <strong>Hello world</strong><br />
      Lorem ipsum dolor sit amet
    </Alert>
  ))
  .add('<Alert.info />', () => (
    <Alert.info onClose={() => {}}>
      <strong>Info</strong><br />
      Lorem ipsum dolor sit amet
    </Alert.info>
  ))
  .add('<Alert.success />', () => (
    <Alert.success onClose={() => {}}>
      <strong>Success</strong><br />
      Lorem ipsum dolor sit amet
    </Alert.success>
  ))
  .add('<Alert.error />', () => (
    <Alert.error onClose={() => {}}>
      <strong>Error</strong><br />
      Lorem ipsum dolor sit amet
    </Alert.error>
  ))
  .add('<Alert variant="info" />', () => (
    <Alert variant="info" onClose={() => {}}>
      <strong>Info</strong><br />
      Lorem ipsum dolor sit amet
    </Alert>
  ))
  .add('<Alert variant="success" />', () => (
    <Alert variant="success" onClose={() => {}}>
      <strong>Success</strong><br />
      Lorem ipsum dolor sit amet
    </Alert>
  ))
  .add('<Alert variant="error" />', () => (
    <Alert variant="error" onClose={() => {}}>
      <strong>Error</strong><br />
      Lorem ipsum dolor sit amet
    </Alert>
  ));

