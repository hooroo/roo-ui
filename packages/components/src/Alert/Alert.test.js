import React from 'react';
import { qantas as theme } from '@roo-ui/themes';
import { shallowWithTheme } from '@roo-ui/test-utils';
import { axe } from 'jest-axe';

import Alert from '.';

describe('<Alert />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallowWithTheme(
      <Alert>Hello world</Alert>,
      theme,
    ).dive();
  });

  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('has no accessibility errors', async () => {
    expect(await axe(wrapper.html())).toHaveNoViolations();
  });

  describe('<Alert.success />', () => {
    beforeEach(() => {
      wrapper = shallowWithTheme(
        <Alert.success>Hello world</Alert.success>,
        theme,
      ).dive();
    });

    it('renders correctly', () => {
      expect(wrapper).toMatchSnapshot();
    });
  });

  describe('<Alert.error />', () => {
    beforeEach(() => {
      wrapper = shallowWithTheme(
        <Alert.error>Hello world</Alert.error>,
        theme,
      ).dive();
    });

    it('renders correctly', () => {
      expect(wrapper).toMatchSnapshot();
    });
  });

  describe('<Alert.info />', () => {
    beforeEach(() => {
      wrapper = shallowWithTheme(
        <Alert.info>Hello world</Alert.info>,
        theme,
      ).dive();
    });

    it('renders correctly', () => {
      expect(wrapper).toMatchSnapshot();
    });
  });

  describe('on close', () => {
    let onClose;

    beforeEach(() => {
      onClose = jest.fn();
      wrapper = shallowWithTheme(
        <Alert onClose={onClose}>Hello world</Alert>,
        theme,
      ).dive();
      wrapper.find('NakedButton').simulate('click');
    });

    it('calls #onClose', () => {
      expect(onClose).toHaveBeenCalled();
    });
  });
});
