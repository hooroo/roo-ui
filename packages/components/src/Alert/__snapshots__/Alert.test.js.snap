// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Alert /> <Alert variant="error" /> renders correctly 1`] = `
<Alert
  mb={3}
  variant="error"
>
  <Component
    mb={3}
    variant="error"
  >
    <Styled(div)
      bg="ui.errorBackground"
      mb={3}
    >
      <div
        className="css-88wsva e1dqxh4r0"
      >
        <Styled(div)
          px={4}
        >
          <div
            className="css-8bw0e3 e1dqxh4r0"
          >
            <Flex
              py={4}
            >
              <div
                className="css-1kfzkx5-Flex e18o73gf0"
              >
                <Styled(div)
                  pr={3}
                >
                  <div
                    className="css-1kdfv5j e1dqxh4r0"
                  >
                    <Icon
                      color="ui.error"
                      name="warning"
                      size={24}
                      title={null}
                    >
                      <Base
                        className="css-ytk37b-Icon e1pa9kzm1"
                        color="ui.error"
                        name="warning"
                        size={24}
                        title={null}
                      >
                        <StyledSvg
                          className="css-ytk37b-Icon e1pa9kzm1"
                          color="ui.error"
                          fill="currentcolor"
                          height={24}
                          title="warning"
                          viewBox="0 0 24 24"
                          width={24}
                        >
                          <svg
                            className="e1pa9kzm1 css-1wizixd-StyledSvg-Icon e1pa9kzm0"
                            color="ui.error"
                            fill="currentcolor"
                            height={24}
                            title="warning"
                            viewBox="0 0 24 24"
                            width={24}
                          >
                            <path
                              d="M1 21h22L12 2 1 21zm12-3h-2v-2h2v2zm0-4h-2v-4h2v4z"
                            />
                          </svg>
                        </StyledSvg>
                      </Base>
                    </Icon>
                  </div>
                </Styled(div)>
                <Text
                  hidden={false}
                  mb={0}
                  textAlign="left"
                  textStyle="text"
                >
                  <span
                    className="css-51vlo9 e1dqxh4r0"
                    hidden={false}
                  >
                    Hello world
                  </span>
                </Text>
              </div>
            </Flex>
          </div>
        </Styled(div)>
      </div>
    </Styled(div)>
  </Component>
</Alert>
`;

exports[`<Alert /> <Alert variant="info" /> renders correctly 1`] = `
<Alert
  mb={3}
  variant="info"
>
  <Component
    mb={3}
    variant="info"
  >
    <Styled(div)
      bg="ui.infoBackground"
      mb={3}
    >
      <div
        className="css-1ocau4u e1dqxh4r0"
      >
        <Styled(div)
          px={4}
        >
          <div
            className="css-8bw0e3 e1dqxh4r0"
          >
            <Flex
              py={4}
            >
              <div
                className="css-1kfzkx5-Flex e18o73gf0"
              >
                <Styled(div)
                  pr={3}
                >
                  <div
                    className="css-1kdfv5j e1dqxh4r0"
                  >
                    <Icon
                      color="greys.charcoal"
                      name="info"
                      size={24}
                      title={null}
                    >
                      <Base
                        className="css-vo8vz7-Icon e1pa9kzm1"
                        color="greys.charcoal"
                        name="info"
                        size={24}
                        title={null}
                      >
                        <StyledSvg
                          className="css-vo8vz7-Icon e1pa9kzm1"
                          color="greys.charcoal"
                          fill="currentcolor"
                          height={24}
                          title="info"
                          viewBox="0 0 24 24"
                          width={24}
                        >
                          <svg
                            className="e1pa9kzm1 css-g2o246-StyledSvg-Icon e1pa9kzm0"
                            color="greys.charcoal"
                            fill="currentcolor"
                            height={24}
                            title="info"
                            viewBox="0 0 24 24"
                            width={24}
                          >
                            <path
                              d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-6h2v6zm0-8h-2V7h2v2z"
                            />
                          </svg>
                        </StyledSvg>
                      </Base>
                    </Icon>
                  </div>
                </Styled(div)>
                <Text
                  hidden={false}
                  mb={0}
                  textAlign="left"
                  textStyle="text"
                >
                  <span
                    className="css-51vlo9 e1dqxh4r0"
                    hidden={false}
                  >
                    Hello world
                  </span>
                </Text>
              </div>
            </Flex>
          </div>
        </Styled(div)>
      </div>
    </Styled(div)>
  </Component>
</Alert>
`;

exports[`<Alert /> <Alert variant="success" /> renders correctly 1`] = `
<Alert
  mb={3}
  variant="success"
>
  <Component
    mb={3}
    variant="success"
  >
    <Styled(div)
      bg="ui.successBackground"
      mb={3}
    >
      <div
        className="css-1ju020a e1dqxh4r0"
      >
        <Styled(div)
          px={4}
        >
          <div
            className="css-8bw0e3 e1dqxh4r0"
          >
            <Flex
              py={4}
            >
              <div
                className="css-1kfzkx5-Flex e18o73gf0"
              >
                <Styled(div)
                  pr={3}
                >
                  <div
                    className="css-1kdfv5j e1dqxh4r0"
                  >
                    <Icon
                      color="ui.success"
                      name="checkCircle"
                      size={24}
                      title={null}
                    >
                      <Base
                        className="css-510x8p-Icon e1pa9kzm1"
                        color="ui.success"
                        name="checkCircle"
                        size={24}
                        title={null}
                      >
                        <StyledSvg
                          className="css-510x8p-Icon e1pa9kzm1"
                          color="ui.success"
                          fill="currentcolor"
                          height={24}
                          title="checkCircle"
                          viewBox="0 0 24 24"
                          width={24}
                        >
                          <svg
                            className="e1pa9kzm1 css-1pn7qtu-StyledSvg-Icon e1pa9kzm0"
                            color="ui.success"
                            fill="currentcolor"
                            height={24}
                            title="checkCircle"
                            viewBox="0 0 24 24"
                            width={24}
                          >
                            <path
                              d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm-2 15l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"
                            />
                          </svg>
                        </StyledSvg>
                      </Base>
                    </Icon>
                  </div>
                </Styled(div)>
                <Text
                  hidden={false}
                  mb={0}
                  textAlign="left"
                  textStyle="text"
                >
                  <span
                    className="css-51vlo9 e1dqxh4r0"
                    hidden={false}
                  >
                    Hello world
                  </span>
                </Text>
              </div>
            </Flex>
          </div>
        </Styled(div)>
      </div>
    </Styled(div)>
  </Component>
</Alert>
`;

exports[`<Alert /> <Alert.error /> renders correctly 1`] = `
<Alert.error
  mb={3}
  variant="error"
>
  <Component
    mb={3}
    variant="error"
  >
    <Styled(div)
      bg="ui.errorBackground"
      mb={3}
    >
      <div
        className="css-88wsva e1dqxh4r0"
      >
        <Styled(div)
          px={4}
        >
          <div
            className="css-8bw0e3 e1dqxh4r0"
          >
            <Flex
              py={4}
            >
              <div
                className="css-1kfzkx5-Flex e18o73gf0"
              >
                <Styled(div)
                  pr={3}
                >
                  <div
                    className="css-1kdfv5j e1dqxh4r0"
                  >
                    <Icon
                      color="ui.error"
                      name="warning"
                      size={24}
                      title={null}
                    >
                      <Base
                        className="css-ytk37b-Icon e1pa9kzm1"
                        color="ui.error"
                        name="warning"
                        size={24}
                        title={null}
                      >
                        <StyledSvg
                          className="css-ytk37b-Icon e1pa9kzm1"
                          color="ui.error"
                          fill="currentcolor"
                          height={24}
                          title="warning"
                          viewBox="0 0 24 24"
                          width={24}
                        >
                          <svg
                            className="e1pa9kzm1 css-1wizixd-StyledSvg-Icon e1pa9kzm0"
                            color="ui.error"
                            fill="currentcolor"
                            height={24}
                            title="warning"
                            viewBox="0 0 24 24"
                            width={24}
                          >
                            <path
                              d="M1 21h22L12 2 1 21zm12-3h-2v-2h2v2zm0-4h-2v-4h2v4z"
                            />
                          </svg>
                        </StyledSvg>
                      </Base>
                    </Icon>
                  </div>
                </Styled(div)>
                <Text
                  hidden={false}
                  mb={0}
                  textAlign="left"
                  textStyle="text"
                >
                  <span
                    className="css-51vlo9 e1dqxh4r0"
                    hidden={false}
                  >
                    Hello world
                  </span>
                </Text>
              </div>
            </Flex>
          </div>
        </Styled(div)>
      </div>
    </Styled(div)>
  </Component>
</Alert.error>
`;

exports[`<Alert /> <Alert.info /> renders correctly 1`] = `
<Alert.info
  mb={3}
  variant="info"
>
  <Component
    mb={3}
    variant="info"
  >
    <Styled(div)
      bg="ui.infoBackground"
      mb={3}
    >
      <div
        className="css-1ocau4u e1dqxh4r0"
      >
        <Styled(div)
          px={4}
        >
          <div
            className="css-8bw0e3 e1dqxh4r0"
          >
            <Flex
              py={4}
            >
              <div
                className="css-1kfzkx5-Flex e18o73gf0"
              >
                <Styled(div)
                  pr={3}
                >
                  <div
                    className="css-1kdfv5j e1dqxh4r0"
                  >
                    <Icon
                      color="greys.charcoal"
                      name="info"
                      size={24}
                      title={null}
                    >
                      <Base
                        className="css-vo8vz7-Icon e1pa9kzm1"
                        color="greys.charcoal"
                        name="info"
                        size={24}
                        title={null}
                      >
                        <StyledSvg
                          className="css-vo8vz7-Icon e1pa9kzm1"
                          color="greys.charcoal"
                          fill="currentcolor"
                          height={24}
                          title="info"
                          viewBox="0 0 24 24"
                          width={24}
                        >
                          <svg
                            className="e1pa9kzm1 css-g2o246-StyledSvg-Icon e1pa9kzm0"
                            color="greys.charcoal"
                            fill="currentcolor"
                            height={24}
                            title="info"
                            viewBox="0 0 24 24"
                            width={24}
                          >
                            <path
                              d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-6h2v6zm0-8h-2V7h2v2z"
                            />
                          </svg>
                        </StyledSvg>
                      </Base>
                    </Icon>
                  </div>
                </Styled(div)>
                <Text
                  hidden={false}
                  mb={0}
                  textAlign="left"
                  textStyle="text"
                >
                  <span
                    className="css-51vlo9 e1dqxh4r0"
                    hidden={false}
                  >
                    Hello world
                  </span>
                </Text>
              </div>
            </Flex>
          </div>
        </Styled(div)>
      </div>
    </Styled(div)>
  </Component>
</Alert.info>
`;

exports[`<Alert /> <Alert.success /> renders correctly 1`] = `
<Alert.success
  mb={3}
  variant="success"
>
  <Component
    mb={3}
    variant="success"
  >
    <Styled(div)
      bg="ui.successBackground"
      mb={3}
    >
      <div
        className="css-1ju020a e1dqxh4r0"
      >
        <Styled(div)
          px={4}
        >
          <div
            className="css-8bw0e3 e1dqxh4r0"
          >
            <Flex
              py={4}
            >
              <div
                className="css-1kfzkx5-Flex e18o73gf0"
              >
                <Styled(div)
                  pr={3}
                >
                  <div
                    className="css-1kdfv5j e1dqxh4r0"
                  >
                    <Icon
                      color="ui.success"
                      name="checkCircle"
                      size={24}
                      title={null}
                    >
                      <Base
                        className="css-510x8p-Icon e1pa9kzm1"
                        color="ui.success"
                        name="checkCircle"
                        size={24}
                        title={null}
                      >
                        <StyledSvg
                          className="css-510x8p-Icon e1pa9kzm1"
                          color="ui.success"
                          fill="currentcolor"
                          height={24}
                          title="checkCircle"
                          viewBox="0 0 24 24"
                          width={24}
                        >
                          <svg
                            className="e1pa9kzm1 css-1pn7qtu-StyledSvg-Icon e1pa9kzm0"
                            color="ui.success"
                            fill="currentcolor"
                            height={24}
                            title="checkCircle"
                            viewBox="0 0 24 24"
                            width={24}
                          >
                            <path
                              d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm-2 15l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"
                            />
                          </svg>
                        </StyledSvg>
                      </Base>
                    </Icon>
                  </div>
                </Styled(div)>
                <Text
                  hidden={false}
                  mb={0}
                  textAlign="left"
                  textStyle="text"
                >
                  <span
                    className="css-51vlo9 e1dqxh4r0"
                    hidden={false}
                  >
                    Hello world
                  </span>
                </Text>
              </div>
            </Flex>
          </div>
        </Styled(div)>
      </div>
    </Styled(div)>
  </Component>
</Alert.success>
`;

exports[`<Alert /> contained renders correctly 1`] = `
<Alert
  contained={true}
  mb={3}
  variant="default"
>
  <Component
    contained={true}
    mb={3}
    variant="default"
  >
    <Styled(div)
      bg="greys.porcelain"
      mb={3}
    >
      <div
        className="css-1cuafi0 e1dqxh4r0"
      >
        <Container
          gutter="default"
          maxWidth="default"
        >
          <div
            className="css-1l14fni-Container e188lj6g0"
          >
            <Flex
              py={4}
            >
              <div
                className="css-1kfzkx5-Flex e18o73gf0"
              >
                <Text
                  hidden={false}
                  mb={0}
                  textAlign="left"
                  textStyle="text"
                >
                  <span
                    className="css-51vlo9 e1dqxh4r0"
                    hidden={false}
                  >
                    Hello world
                  </span>
                </Text>
              </div>
            </Flex>
          </div>
        </Container>
      </div>
    </Styled(div)>
  </Component>
</Alert>
`;

exports[`<Alert /> renders correctly 1`] = `
<Alert
  mb={3}
  variant="default"
>
  <Component
    mb={3}
    variant="default"
  >
    <Styled(div)
      bg="greys.porcelain"
      mb={3}
    >
      <div
        className="css-1cuafi0 e1dqxh4r0"
      >
        <Styled(div)
          px={4}
        >
          <div
            className="css-8bw0e3 e1dqxh4r0"
          >
            <Flex
              py={4}
            >
              <div
                className="css-1kfzkx5-Flex e18o73gf0"
              >
                <Text
                  hidden={false}
                  mb={0}
                  textAlign="left"
                  textStyle="text"
                >
                  <span
                    className="css-51vlo9 e1dqxh4r0"
                    hidden={false}
                  >
                    Hello world
                  </span>
                </Text>
              </div>
            </Flex>
          </div>
        </Styled(div)>
      </div>
    </Styled(div)>
  </Component>
</Alert>
`;
